src/core/init.js

```diff
@@ -11,37 +11,22 @@ geo.fileReaders = {};
  * Convenient function to define JS inheritance
  */
 //////////////////////////////////////////////////////////////////////////////
-inherit = function (C, P) { // jshint ignore: line
+geo.inherit = function (C, P) { // jshint ignore: line
   "use strict";
 
-  var F = function () {
-  };
+  var F = inherit.func();
   F.prototype = P.prototype;
   C.prototype = new F();
-  C.uber = P.prototype;
   C.prototype.constructor = C;
 };
-
-//////////////////////////////////////////////////////////////////////////////
-/**
- * Convenient function to get size of an object
- *
- * @param {object} obj
- * @returns {number} *
- */
-//////////////////////////////////////////////////////////////////////////////
-Object.size = function (obj) { // jshint ignore: line
+geo.inherit.func = function () {
```

Comment:
Right. For others, after our discussions we think that extending base objects may not be as bad (the JS community has different opinions on this) but we will avoid doing so as much as possible and if we do, we will use underscore to mark it special. Also, we discussed about using underscore library in the future which I think is a low priority for now.